models/MFCStack.py

```diff
@@ -50,5 +50,7 @@ def __init__(
             Linear(25, output_dim),
```

Comment:
Same here. self.mlp is part of the head part

Reply:
Fixed